prow/e2e-suite.sh

```diff
@@ -77,6 +77,7 @@ make init
 
 trap cleanup EXIT
 get_resource "${RESOURCE_TYPE}" "${OWNER}" "${INFO_PATH}" "${FILE_LOG}"
+test_cluster
```

Comment:
it seems conceptually confusing to test before setup..  did you mean it? =)

Reply:
renamed to check_cluster